internal/jimm/access.go

```diff
@@ -717,14 +717,14 @@ func (j *JIMM) CountGroups(ctx context.Context, user *openfga.User) (int, error)
 	return count, nil
 }
 
-// GetGroup returns a group based on the provided UUID.
-func (j *JIMM) GetGroupByID(ctx context.Context, user *openfga.User, uuid string) (*dbmodel.GroupEntry, error) {
+// GetGroup returns a group based on the provided UUID or name.
+func (j *JIMM) GetGroup(ctx context.Context, user *openfga.User, uuid, name string) (*dbmodel.GroupEntry, error) {
```

Comment:
I suggest a separate method, so we have `GetGroupByID` and `GetGroupByName`

Reply:
Needed to sleep on this but at the end I still wasn't sure about this, but both ways are fine. 
Just noticed the op for the function is set to AddGroup too so one more change.

Reply:
also nitpick: `GetGroupByUUID` and `GetGroupByName`